spec/system/planning_applications/make_public_spec.rb

```diff
@@ -39,12 +39,16 @@
 
     expect(page).to have_content("Public on BOPS Public Portal: Yes")
 
+    expect(planning_application.reload.published_at).not_to be_nil
```

Comment:
Can't we just set this expectation to the time we travel to in this block `"2022-01-01"`